src/UserRepository.js

```diff
@@ -1,7 +1,32 @@
-class UserRepository {
-  constructor() {
+const data = require('../data/users.js');
+const userData = data.testData;
 
+class UserRepository {
+  constructor(userData) {
+    this.data = userData;
+  }
+  getUserData(id) {
+    let userInfo = this.data.find(user => {
+      if (user.id === id) {
+        return user;
+      }
+    });
+    return userInfo;
```

Comment:
the find() method's callback needs to return a boolean, so it might be better to just use:
`getUserData(id) {
  return this.data.find(user => user.id === id);
}`
arrow functions often have an implicit return, so the statement `user.id === id` evaluates to true or false and the callback will return that value

Reply:
oooo neat. adjusting now

Reply:
so using arrow functions implies a return so I dont have to have an if block.

essentially 'return (conditional)'

Reply:
yeah, essentially